patches/uClibc-ng/1.0.12/100-provide-_obstack_free.patch

```diff
@@ -0,0 +1,12 @@
+diff -ur uClibc-ng-1.0.12.orig/libc/misc/gnu/obstack.c uClibc-ng-1.0.12/libc/misc/gnu/obstack.c
+--- uClibc-ng-1.0.12.orig/libc/misc/gnu/obstack.c	2016-03-13 15:08:44.408962824 -0700
++++ uClibc-ng-1.0.12/libc/misc/gnu/obstack.c	2016-03-13 15:13:30.129322998 -0700
+@@ -385,7 +385,7 @@
+     abort ();
+ }
+ 
+-# if 0
++# if 1
+ /* Older versions of libc used a function _obstack_free intended to be
+    called by non-GCC compilers.  */
+ strong_alias (obstack_free, _obstack_free)
```

Comment:
Didn't see wbx on IRC in the past few days. The patch should be applicable to 1.0.13 as is, the code hasn't changed between .12 and .13 releases.
Ok to commit? If wbx objects, we can always re-do it in another way later (so long as it does not **break** the samples, :grin:)